test/e2e/must-gather/must_gather_test.go

```diff
@@ -133,8 +136,11 @@ var _ = ginkgo.Describe("[must-gather] NRO data collected", func() {
 					podFolderNames = append(podFolderNames, podFolder.Name())
 				}
 				gomega.Expect(podFolderNames).To(gomega.ContainElement(gomega.MatchRegexp("^numaresources-controller-manager*")))
-				gomega.Expect(podFolderNames).To(gomega.ContainElement(gomega.MatchRegexp("^numaresourcesoperator-mcp-test*")))
 				gomega.Expect(podFolderNames).To(gomega.ContainElement(gomega.MatchRegexp("^secondary-scheduler*")))
+
+				if configuration.Plat == platform.Kubernetes {
```

Comment:
that's ok, but thinking about it, must-gather is supported only on OCP platform; so the must-gather e2e tests should just skip (all of them) if running on non-OCP platform.

Reply:
Ok